SPECS/glibc/glibc.spec

```diff
@@ -145,14 +141,16 @@ export CFLAGS
 export CXXFLAGS
 
 cd %{_builddir}/%{name}-build
+echo "rootsbindir=/usr/sbin" > configparms
 ../%{name}-%{version}/configure \
         --prefix=%{_prefix} \
-        --disable-profile \
         --disable-werror \
-        --enable-kernel=3.2 \
-        --enable-bind-now \
+        --enable-kernel=4.14 \
+        --enable-stack-protector=strong        \
+        --with-headers=/usr/include            \
+        libc_cv_slibdir=/usr/lib \
```

Comment:
nit:  maybe set this macro in a place that doesn't interrupt the flow of --enable and --disable. .

Reply:
will fix